common/views/pages/_app.js

```diff
@@ -87,6 +107,16 @@ export default class WecoApp extends App {
       titleCase: false
     }]);
 
+    try {
+      if (document.hidden) {
+        pageVisibilityLastChanged = window.performance.now(); // in case page is opened in a new tab
+      }
+      document.addEventListener('visibilitychange', calculateHiddenTimeOnPage, false);
```

Comment:
Is there a reason to set the useCapture here to false, where that's the default?